e2e-tests/playwright/e2e/plugins/topology/topology.spec.ts

```diff
@@ -15,11 +15,34 @@ test.describe('Test Topology Plugin', () => {
     await common.loginAsGuest();
   });
 
-  test('Verify pods visibility in the Topology tab', async () => {
+  test('Verify pods visibility in the Topology tab', async ({ page }) => {
     await catalog.goToBackstageJanusProject();
     await uiHelper.clickTab('Topology');
     await uiHelper.verifyText('backstage-janus');
     await uiHelper.verifyText('rhdh');
     await uiHelper.verifyText('rhdh-rbac');
+    const url_button = page
+      .locator('g')
+      .filter({ hasText: /Open URL/ })
+      .getByRole('button')
+      .first();
+    expect(await url_button.getAttribute('href')).toContain(
+      'https://rhdh-backstage-showcase',
+    );
+    await page
+      .locator('g')
+      .filter({ hasText: /Open URL/ })
+      .locator('image')
+      .first()
+      .click();
+    await page.getByLabel('Pod').click();
+    await page.getByLabel('Pod').getByText('1', { exact: true }).click();
+    await uiHelper.clickTab('Details');
+    await uiHelper.isTextVisible('Status');
+    await uiHelper.isTextVisible('Active');
```

Comment:
```suggestion
    await uiHelper.verifyText('Status');
    await uiHelper.verifyText('Active');
```
`isTextVisible` doesn't do assertion instead use `verifyText`